package.json

```diff
@@ -37,11 +39,15 @@
     "@babel/preset-env": "^7.4.2",
     "@babel/preset-react": "^7.0.0",
     "babel-loader": "^8.0.5",
+    "enzyme": "^3.9.0",
```

Comment:
Quick question--are you able to run your jest code on ES6+ syntax and have it work? You may need to have one more piece that connects babel and jest (I'd also recommend looking in to the eslint plug-in for jest!).

Reply:
I was using fat arrow functions, so I believe it was working with ES6? Though I just installed the mentioned modules and plugins as well, thanks.